7.2/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM php:7.2-fpm
+
+RUN apt-get update
+
+RUN apt-get install -y libz-dev libmemcached-dev
+RUN pecl install memcached-3.0.3
+RUN echo extension=memcached.so >> /usr/local/etc/php/conf.d/memcached.ini
+
+# gd extension
+RUN apt-get install -y \
+  libpng-dev \
+  libfreetype6-dev \
+  libjpeg-dev \
+  libxpm-dev \
+  libxml2-dev \
+  libwebp-dev  # php >=7.0 (use libvpx for php <7.0)
+RUN docker-php-ext-configure gd \
+    --with-freetype-dir=/usr/include/ \
+    --with-jpeg-dir=/usr/include/ \
+    --with-xpm-dir=/usr/include \
+    --with-webp-dir=/usr/include/ # php >=7.0 (use libvpx for php <7.0)
+RUN docker-php-ext-install gd
+
+RUN docker-php-ext-install zip
+
+RUN apt-get install -y libicu-dev
+RUN docker-php-ext-install intl
+
+RUN docker-php-ext-install mysqli
+RUN docker-php-ext-install pdo_mysql
+RUN docker-php-ext-install opcache
+RUN docker-php-ext-install bcmath
+
+RUN pecl install xdebug
+RUN echo "zend_extension=`php -i | grep ^extension_dir | cut -f 3 -d ' '`/xdebug.so" >> /usr/local/etc/php/conf.d/xdebug.ini
+
+RUN docker-php-ext-install soap
+
+ENV COMPOSER_HOME=/composer
+RUN curl -sS https://raw.githubusercontent.com/composer/getcomposer.org/1b137f8bf6db3e79a38a5bc45324414a6b1f9df2/web/installer | php -- --quiet && mv composer.phar /usr/local/bin/composer
```

Comment:
we are using this `COPY --from=composer:1.5 /usr/bin/composer /usr/bin/composer`, but you can also use:
`COPY --from=composer:latest /usr/bin/composer /usr/bin/composer`.

In this case, you don't need to care about verifying the signature

Reply:
Good idea: But please with a static composer version. Can you prepare a PR for this? @OskarStark

Reply:
Here you go #13